File: robocode2-gui/src/main/kotlin/net/robocode2/gui/model/BotResults.kt
package net.robocode2.gui.model

import net.robocode2.gui.model.Content
import net.robocode2.gui.model.ContentType

class BotResults(
        id: Int,
        rank: Int,
        survival: Int,
        lastSurvivorBonus: Int,
        bulletDamage: Int,
        bulletKillBonus: Int,
        ramDamage: Int,
        ramKillBonus: Int,
        totalScore: Int,
        firstPlaces: Int,
        secondPlaces: Int,
        thirdPlaces: Int
) : Content(type = ContentType.BOT_RESULTS.type)

File: bot-api/python/tankroyale/botapi/schemas/python/BotResults.py
from dataclasses import dataclass


@dataclass
class BotResults:
    rank: int
    survival: float
    lastSurvivorBonus: float
    bulletDamage: float
    bulletKillBonus: float
    ramDamage: float
    ramKillBonus: float
    totalScore: float
    firstPlaces: int
    secondPlaces: int
    thirdPlaces: int
